tests/Oryx.Tests.Common/StorageAccountEnvVariableForDb.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Oryx.BuildScriptGenerator.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Oryx.Tests.Common
+{
+    public static class StorageAccountEnvVariableForDb
+    {
+        public static List<EnvironmentVariable> AddStorageAccountEnvironmentVariables(List<EnvironmentVariable> envVarList)
```

Comment:
nit: however you end up refactoring this method (I agree with Paul's comment about this being an extension method), let's make sure to add a doc comment to the method so callers are aware of the method's use and parameter